### DISCUSSION TITLE
Apache

### DISCUSSION THREAD

**Discussion by asuri0n** [CleverCloud/terraform-provider-clevercloud#45] [General]
Hello CC team ! :)
do you have a release date for the apache application and filesystem addon ?

**Comment by miton18** [comment]
Soon ;-)
https://github.com/CleverCloud/terraform-provider-clevercloud/pull/46

**Comment by miton18** [comment]
https://registry.terraform.io/providers/CleverCloud/clevercloud/latest/docs/resources/static
